tenant/http_server_user.go

```diff
@@ -67,20 +66,8 @@ func NewHTTPUserHandler(log *zap.Logger, userService influxdb.UserService, passw
 	return svr
 }
 
-type resourceHandler struct {
-	prefix string
-	*UserHandler
-}
-
-func (h *resourceHandler) Prefix() string {
-	return h.prefix
-}
-func (h *UserHandler) MeResourceHandler() *resourceHandler {
-	return &resourceHandler{prefix: prefixMe, UserHandler: h}
-}
-
-func (h *UserHandler) UserResourceHandler() *resourceHandler {
-	return &resourceHandler{prefix: prefixUsers, UserHandler: h}
+func (h *UserHandler) Prefix() string {
+	return prefixUsers
```

Comment:
You could move the `const` declaration in here to reduce its visibility.

Reply:
Should the other uses of the consts be these method calls, or is the containing object not available there?